core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
```

Comment:
Instead of inlining the whole method body I'd like to have an early return here by flipping the condition.